internal/oci/runtime_vm.go

```diff
@@ -814,7 +817,10 @@ func (r *runtimeVM) wait(ctx context.Context, ctrID, execID string) (int32, erro
 		ExecID: execID,
 	})
 	if err != nil {
-		return -1, errdefs.FromGRPC(err)
+		if !errors.Is(err, ttrpc.ErrClosed) {
+			return -1, errdefs.FromGRPC(err)
+		}
+		return -1, errdefs.ErrNotFound
```

Comment:
Should we define a custom error here instead?

Reply:
I'm really on the fence about this.

Defining a custom error seems the cleanest possible way to proceed, but we need to ensure it'll be handled inside our own code.  The reason for that is containerd also returning `errdefs.ErrNotFound` in case the error is `ttrpc.ErrClosed`, which makes me think that it's some kind of non-written convention.

Would you be okay with a follow-up PR where I'd switch the ErrNotFound by our own defined error, carefully taking into consideration the places where we actually can do that?

Reply:
Sounds good :+1: